src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/ParallelThreadsRunner.kt

```diff
@@ -446,6 +453,17 @@ internal open class ParallelThreadsRunner(
         super.close()
         executor.close()
     }
+
+    private inline fun executeInIgnoredSection(iThread: Int, action: () -> Unit) {
+        if (strategy is ManagedStrategy) {
+            strategy.enterIgnoredSection(iThread)
```

Comment:
Do we really need to pass `iThread` here? It should always be `ManagedStrategy.currentThread`.